docs-generation/tests/test_translation_fixes.py

```
@@ -0,0 +1,197 @@
+
+    def test_separator_like_row_does_not_count_as_data_row(self):
+        translated = (
+            "## 输出\n\n"
+            "| 名称 | 数据类型 | 描述 |\n"
+            "|------|-----------|-------------|\n"
+            "| `正向` | CONDITIONING | 正向条件 |\n"
+            "| `负向` | CONDITIONING | 负向条件 |\n"
+            "| `潜空间` | LATENT | 去噪后的潜空间 |\n"
+        )
+        out = btd._fix_output_names_in_translation(translated, self.EN_DOC)
+        rows = [l for l in out.split("\n") if l.strip().startswith("| `")]
```

Ruff (0.16.1)

[error] Ambiguous variable name: `l`

(E741)